### DISCUSSION TITLE
Events throw error in PixiJs version 7.2.0 after following the example

### DISCUSSION THREAD

**Discussion by ThomasvanHoutum** [pixijs/pixijs#9272] [Q&A]
Hi all,
I am trying out PixiJs for the first time and when I was trying out the dragging example (https://pixijs.io/examples/#/events/dragging.js) I get a bunch of errors refering to the events: [Console Errors](https://i.stack.imgur.com/yFR0L.png)

I am testing it out in a Vue 3 project, as that is where I want to implement it eventually.

I tried replacing the `bunny.interactive = true;` to `img.eventMode = 'static'` but it gives me the same error, but at least the PixiJs warning is gone.

How I call the start event `img.on('pointerdown', this.onDragStart, img);`

And the other events:

```
this.app.stage.on('pointerup', this.onDragEnd);
this.app.stage.on('pointerupoutside', this.onDragEnd);
```
Why does it give me these errors when I've followed what is described on the example page? Perhaps I am just missing something simple but I can't figure it out.

Thanks in advance.

**Comment by SuperSodaSea** [comment]
Would you like to show your own code here, or create a repo on GitHub that can reproduce your issue?
